inference-engine/tests/functional/plugin/cpu/single_layer_tests/cum_sum.cpp

```diff
@@ -0,0 +1,250 @@
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<cumSumParams> obj) {
+        ngraph::element::Type inputPrecision;
+        std::pair<std::vector<ngraph::PartialShape>, std::vector<std::vector<ngraph::Shape>>> shapes;
+        std::int64_t axis;
+        bool exclusive;
+        bool reverse;
+        std::tie(inputPrecision, shapes, axis, exclusive, reverse) = obj.param;
+
+        std::ostringstream result;
+        result << inputPrecision << "_" << "IS=" << CommonTestUtils::partialShape2str(shapes.first) << "_" << "TS=";
+        for (const auto& shape : shapes.second) {
+            result << "(";
+            for (const auto& item : shape) {
+                result << CommonTestUtils::vec2str(item) << "_";
+            }
+            result << ")_";
+        }
+
+        result << "Axis=" << axis << "_" << (exclusive ? "exclusive" : "") << "_" << (reverse ? "reverse" : "");
+        return result.str();
+    }
+
+protected:
+    void SetUp() override {
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+        ngraph::element::Type inputPrecision;
+        std::pair<std::vector<ngraph::PartialShape>, std::vector<std::vector<ngraph::Shape>>> shapes;
+        std::int64_t axis;
+        bool exclusive;
+        bool reverse;
+        std::tie(inputPrecision, shapes, axis, exclusive, reverse) = this->GetParam();
+
+        for (size_t i = 0; i < shapes.second.size(); i++) {
+            targetStaticShapes.push_back(shapes.second[i]);
+        }
+        inputDynamicShapes = shapes.first;
+
+        auto params = ngraph::builder::makeDynamicParams(inputPrecision, { inputDynamicShapes.front() });
+        auto axisNode = ngraph::opset1::Constant::create(ngraph::element::i32, ngraph::Shape{}, std::vector<int64_t>{axis})->output(0);
+        auto cumSum = ngraph::builder::makeCumSum(params[0], axisNode, exclusive, reverse);
+
+        function = std::make_shared<ngraph::Function>(ngraph::NodeVector{ cumSum }, params, "CumSumLayerCPUTest");
+        functionRefs = ngraph::clone_function(*function);
+    }
+};
+
+TEST_P(CumSumLayerCPUTest, CompareWithRefs) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+
+    run();
+    // TODO: Should be uncommented after updating the CheckPluginRelatedResults() method
+    //CheckPluginRelatedResults(executableNetwork, "CumSum");
+}
+
+const ngraph::element::TypeVector inputPrecision = {
+    ngraph::element::i8,
+    ngraph::element::u8,
+    ngraph::element::i16,
+    ngraph::element::i32,
+    ngraph::element::f32
+};
+
+const std::vector<int64_t> axes = { 0, 1, 2, 3, 4, 5, 6 };
+const std::vector<int64_t> negativeAxes = { -1, -2, -3, -4, -5, -6 };
+
+const std::vector<bool> exclusive = { true, false };
+const std::vector<bool> reverse = { true, false };
+
+using shapesSuite = std::vector<std::pair<std::vector<ngraph::PartialShape>, std::vector<std::vector<ngraph::Shape>>>>;
+
+shapesSuite inShapes = {
+    {
+        // dynamic
+        {
+            {-1}
```

Comment:
add test cases with upper bounds

Reply:
Test cases with lower/upper bounds was added, thank you.